django/src/rdwatch/server/settings.py

```diff
@@ -93,6 +94,13 @@ class BaseConfiguration(Configuration):
     CELERY_RESULT_EXTENDED = True
     CELERYD_TIME_LIMIT = 1200
 
+    CELERY_BEAT_SCHEDULE = {
+        'delete-temp-model-runs-beat': {
+            'task': 'rdwatch.tasks.delete_temp_model_runs_task',
+            'schedule': timedelta(seconds=60),
```

Comment:
for a dummy task do we want to bump this up to like once an hour or day (outside of testing?).  Maybe a configurable time that changes based on deployment/development?

Reply:
I changed it to an hour for now. But yeah we need to decide on the exact semantics of the "expiration time" of a model run, and adjust the interval accordingly.